samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -50,7 +50,7 @@ spec:
     spec:
       containers:
       - name: details
-        image: istio/examples-bookinfo-details-v1:1.14.0
+        image: "{{ .Values.hub }}/istio/examples-bookinfo-details-v1:1.14.0"
```

Comment:
I think `"{{ .Values.hub }}/examples-bookinfo-details-v1:1.14.0"` - istio is part of the hub